caster-front/src/router/index.ts

```diff
@@ -16,6 +17,11 @@ const router = createRouter({
       name: "debug",
       component: DebugPlayer,
     },
+    {
+      path: "/gpsError",
```

Comment:
isn't the naming convention for URLs to use `kebab-case` instead of `camelCase` ? :) Fine with me though.

Reply:
fixed to kebab-case